installation/manual-installation/ubuntu/README.md

```diff
@@ -72,7 +72,7 @@ tar -xzf /tmp/rocket.chat.tgz -C /tmp
 Install (this guide uses /opt but feel free to choose a different directory):
 
 ```bash
-cd /tmp/bundle/programs/server && npm install
+cd /tmp/bundle/programs/server && sudo npm install
```

Comment:
There should be no `sudo` necessary, since the modules will be installed into the current working directory - which then will be changed to `rocketchat:rocketchat` later anyways.